_topic_maps/_topic_map_rosa.yml

```
@@ -153,6 +153,8 @@ Topics:
   File: rosa-hcp-sts-creating-a-cluster-quickly
 - Name: Using the Node Tuning Operator on ROSA with HCP
   File: rosa-tuning-config
+- Name: Creating an AWS KMS key for customer encryption
+  File: rosa-hcp-creating-cluster-with-aws-kms-key
```

```suggestion
- Name: Creating ROSA with HCP clusters using a custom AWS KMS encryption key
  File: rosa-hcp-creating-cluster-with-aws-kms-key
- Name: Using the Node Tuning Operator on ROSA with HCP
  File: rosa-tuning-config
```

Just suggest changing this to a "Creating ROSA with HCP clusters" type of document, and moving it right under the "using the default options" assembly

---

@wgordon17 Moved it but not sure what you mean by "Just suggest changing this to a "Creating ROSA with HCP clusters" type of document"?

---

@fmcdonal You need to click on the "Commit suggestion" so that my change gets applied.